code/modules/clothing/chameleon.dm

```diff
@@ -156,7 +156,7 @@
 		var/obj/item/I = target
 
 		I.item_state = initial(picked_item.item_state)
-		I.item_color = initial(picked_item.item_color)
```

Comment:
This line is also required, dont remove it

Reply:
Alr.

Reply:
It's functional after testing.